packages/axes/src/inputType/WheelInput.ts

```diff
@@ -130,7 +142,7 @@ export class WheelInput implements InputType {
   }
 
   private _onWheel(event: WheelEvent) {
-    if (!this._enabled) {
+    if (!this._enabled || !isValidKey(event, this.options.inputKey)) {
```

Comment:
The wheel will have to be careful.

Multi-touch in the browser works with ctrl + wheel.

Reply:
Since ctrl + wheel also has a zoom function in the browser, it looks like it should be handled with `preventDefault`.
In Axes, `preventDefault` is already applied for ctrl + wheel, can be there any additional processing needed?